test/integration/data-grid/fixture-custom/toolpad/components/Other.tsx

```diff
@@ -0,0 +1,27 @@
+import { createComponent } from '@mui/toolpad/browser';
+import * as React from 'react';
+
+export interface Props {
```

Comment:
I guess these Props and argTypes still need to be defined here even though they're not used? Just making sure.

Reply:
Not really necessary, I just copied the docs